src/main/resources/public/js/src/launches/historyGrid/HistoryItemCellView.js

```diff
@@ -123,8 +123,47 @@ define(function (require) {
                 }
             }
         },
+        getUrl: function () {
```

Comment:
Try to cunstruct url with using pathNames field of the item model(this.model) instead of current url parsing.